adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -634,6 +634,11 @@ private void verifyBrokerRedirectUri(final AuthenticationRequest request) throws
             throw new UsageAuthenticationException(ADALError.DEVELOPER_REDIRECTURI_INVALID, "The redirectUri is null or blank.");
         }
 
+        if (inputUri.equalsIgnoreCase(AuthenticationConstants.Broker.BROKER_REDIRECT_URI))
+        {
```

Comment:
Nit: our bracket-style is Egyptian

Can we add a log statement here to capture the early-return?